jasper_test.go

```diff
@@ -39,27 +39,32 @@ const taskTimeout = 5 * time.Second
 
 func trueCreateOpts() *CreateOptions {
 	return &CreateOptions{
-		Args: []string{"true"},
+		Args:   []string{"true"},
+		Output: OutputOptions{},
```

Comment:
given that this is not a pointer it should be fine let the default constructor take care of it. are you still seeing an error without this line (and friends?)

Reply:
I removed OutputOptions{} since it's not needed.